src/com/ableton/PythonPackageVersionReader.groovy

```diff
@@ -0,0 +1,56 @@
+package com.ableton
+
+
+/**
+ * Class to get the version of a Python package by parsing the file that sets it.
+ */
+class PythonPackageVersionReader implements Serializable {
```

Comment:
Given that the library is called `python-utils`, I think the class could simply be called `PackageVersionReader`. What do you think about this?

Reply:
I was concerned with the resulting singleton -- suddenly you have `packageVersionReader` which could be any kind of language package...

Reply:
We can always name the singleton something else.

Reply:
But don't we take the risk of having some naming clash? For instance, what if I were to create a `PackageVersionReader` in `rust-utils` and then import both `python-utils` and `rust-utils`?

Reply:
@nre-ableton how about renaming it to `PythonPackage`, with a method `readVersion()`? That way it's nice and short, and we avoid collisions. In the future, we might be able to use `PythonPackage` to manipulate or get other kinds of information from the package. What do you think?

Reply:
This has been addressed in bdba347.